catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/model/JsonTransformVisitor.java

```diff
@@ -0,0 +1,172 @@
+  @Override
+  public void visitString(VisitableXmlElement<String> visitable) {
+    JAXBElement<String> element = visitable.getElement();
+    LOGGER.debug(element.getName().getLocalPart());
+    LOGGER.debug(element.getValue());
+
+    builder.setProperty(element.getValue());
+  }
+
+  @Override
+  public void visitLiteralType(VisitableXmlElement<LiteralType> visitable) {
+    JAXBElement<LiteralType> element = visitable.getElement();
+    LOGGER.debug(element.getName().getLocalPart());
+
+    List<Serializable> values = element.getValue().getContent();
+    if (values == null || values.isEmpty()) {
+      LOGGER.debug("No values found on literal type");
+      return;
+    }
+
+    // Assumption: we only support one literal value
+    values.forEach(v -> LOGGER.debug((String) v));
+    builder.setValue(values.get(0).toString());
+  }
+
+  @Override
+  public void visitFunctionType(VisitableXmlElement<FunctionType> visitable) {
+    JAXBElement<FunctionType> element = visitable.getElement();
+    LOGGER.debug(element.getName().getLocalPart());
+
+    FunctionType functionType = element.getValue();
+    String functionName = functionType.getName();
+
+    if (FORMS_FUNCTION_V1.equals(functionName)) {
+      List<Optional<Serializable>> args =
+          functionType
+              .getExpression()
+              .stream()
+              .map(JAXBElement::getValue)
+              .map(LiteralType.class::cast)
+              .flatMap(
+                  t ->
+                      (t.getContent() == null || t.getContent().isEmpty())
+                          ? Stream.of(Optional.<Serializable>empty())
+                          : t.getContent().stream().map(Optional::of))
+              .collect(Collectors.toList());
+
+      args.forEach(o -> LOGGER.debug(o.toString()));
+
+      Function<Serializable, Boolean> boolFunc = s -> Boolean.parseBoolean((String) s);
+      builder.setTemplatedValues(
+          get(args, 0, String.class), /* Default Value */
+          get(args, 1, String.class), /* Node ID */
+          get(args, 2, boolFunc), /* Is Visible */
+          get(args, 3, boolFunc)); /* Is Read-only */
+
+    } else {
+      throw new FilterProcessingException("Could not find a valid function name");
+    }
+  }
+
+  @Override
+  public void visitBinaryLogicType(VisitableXmlElement<BinaryLogicOpType> visitable) {
+    JAXBElement<BinaryLogicOpType> element = visitable.getElement();
+    String localPart = element.getName().getLocalPart();
+    LOGGER.debug(localPart);
+
+    builder.beginBinaryLogicType(localPart.toUpperCase());
+    element.getValue().getOps().forEach(jax -> makeVisitable(jax).accept(this));
+    builder.endBinaryLogicType();
+  }
+
+  @Override
+  public void visitBinaryComparisonType(VisitableXmlElement<BinaryComparisonOpType> visitable) {
+    JAXBElement<BinaryComparisonOpType> element = visitable.getElement();
+    String localPart = element.getName().getLocalPart();
+    LOGGER.debug(localPart);
```

Comment:
There are a few of these logging statements in this class that could possibly be cleaned up